### ISSUE TITLE
Uneven menu layout in pop-up in Firefox version

### ISSUE THREAD

**Issue by TotalCaesar659** [ray-lothian/UserAgent-Switcher#105]
Menu layout in pop-up in Firefox version is uneven (Lubuntu 20.04.1, Firefox 81.0).
![1](https://user-images.githubusercontent.com/14265316/95479199-2aedc480-0993-11eb-9228-9f0956d75f17.png)
Chromium one is alright.
![2](https://user-images.githubusercontent.com/14265316/95479225-317c3c00-0993-11eb-8ae3-4da1bc94075f.png)

**Comment by JonnyTech** [comment]
It is fine for me on Debian 10 Firefox. Any font scaling in use? Tried disabling all other addons?

![ua-nq8](https://user-images.githubusercontent.com/3584732/95493010-c7b06200-0993-11eb-957b-ce0c780cd91c.png)

**Comment by TotalCaesar659** [comment]
I removed all caches and configs and installed [Mozilla](https://addons.mozilla.org/en-US/firefox/addon/user-agent-string-switcher/) version, same result as above.
But when I removed everything again and install github version in about:debugging, it started to look alright.
![0](https://user-images.githubusercontent.com/14265316/95497840-a491ac80-09ab-11eb-8610-59611fd7c40b.png)
I have a doubt that I've touched any font settings. Perhaps Mozilla version doesn't suit Lubuntu specifically.
